docs/Post-Action-Registry.md

```diff
@@ -156,6 +156,7 @@ Opens a file in the editor. For command line cases this post action will be igno
  - **Specific Configuration** :
    - `args`:
      - `primaryOutputIndexes` (string): A semicolon delimited list of indexes to the primary outputs. Note: If primary outputs are conditional, multiple post actions with the same conditions as the primary outputs might be necessary.
+     - `solutionFolder` (string) (optional) (supported in .NET 5.0.200 or later): the destination solution folder path to add the projects to. 
```

Comment:
@KathleenDollard could you please also review doc change?

How we should deal with doc changes for new features? 
Should we only make them available on wiki once the feature is released?
What is the correct way to mention the version since when feature is available? Are we sure about 5.0.200 or still discussing whether we make another template engine release with SDK 5.0.200?